packages/postgres-database/src/simcore_postgres_database/models/comp_pipeline.py

```diff
@@ -10,6 +10,13 @@
 
 
 class StateType(enum.Enum):
+    """Discrete states in a task lifecycle
+
+    SEE RunningState for a project's execution state
```

Comment:
these states are the exact same ones as the models-library. ideally they should be exactly the same. I am not sure why you specify a different comment in the docstring?

Reply:
OK. I will add a test to make sure Enum sync definitions are in order